cmd/client-errors.go

```diff
@@ -214,3 +214,27 @@ type SameFile struct {
 func (e SameFile) Error() string {
 	return fmt.Sprintf("'%s' and '%s' are the same file", e.Source, e.Destination)
 }
+
+// RetentionAlreadyExists - is thrown when we don't want to overwrite existing retention config
+type RetentionAlreadyExists struct {
```

Comment:
```suggestion
type ErrRetentionAlreadyExists struct {
```

Reply:
@klauspost this doesn't follow the existing style, maybe we can do that in another PR

Reply:
It follows all other styles. No need to introduce exported types that will be renamed.